### PR TITLE
Config Updates for GENIE

### PR DESCRIPTION
**Pull request by averyniceday**
- Added PV + PVC for VEP cache
- New arguments/images for genome-nexus server

### CODE REVIEW

File: genome-nexus/vep/genie_gn_vep.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    metadata:
+      creationTimestamp: null
+      labels:
+        run: genie-gn-vep
+    spec:
+      containers:
+      - env:
+        - name: SERVER_PORT
+          value: "8888"
+        image: genomenexus/gn-vep:gn-vep-beta
```
Comment (ao508) [inline genome-nexus/vep/genie_gn_vep.yaml:33]: @averyniceday the updated docker file has been pushed to `genomenexus/gn-vep:master` so it's okay to update the image and tag to this now

File: genome-nexus/gn_genie.yaml [hunk 2]
```diff
@@ -24,22 +24,24 @@ spec:
...
+        run: genome-nexus-genie
     spec:
       containers:
       - env:
         - name: SERVER_PORT
           value: "8888"
         - name: MONGODB_URI
-          value: mongodb://gn-mongo-v0dot9-genie-mongodb:27017/annotator?connectTimeoutMS=120000
-        image: genomenexus/gn-spring-boot:v1.0.0-263-g387164e
+          value: mongodb://genie-gn-mongo-v0dot9-mongodb:27017/annotator?connectTimeoutMS=120000
```
Comment (sheridancbio) [inline genome-nexus/gn_genie.yaml:34]: the reference genie-gn-mongo-v0dot9-mongodb is now out of date. We plan to update this in a follow up PR

File: genome-nexus/gn_genie.yaml [hunk 3]
```diff
@@ -24,22 +24,24 @@ spec:
...
+        image: genomenexus/gn-spring-boot:genie-gn-spring-boot-beta
```
Comment (sheridancbio) [inline genome-nexus/gn_genie.yaml:35]: we want to adjust this to a production reference in a follow up PR

File: genome-nexus/vep/gn_vep_pv.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```
Comment (sheridancbio) [inline genome-nexus/vep/gn_vep_pv.yaml:1]: This file will not be needed for the kind of persistent volume claim we settled on using. (not using NFS any more)
We will make this correction in a follow up PR.

**Pull request merged**